.travis_coveragerc

```diff
@@ -24,6 +24,8 @@ omit =
     */tests/p*
     */tests/r*
     */tests/s*
+    # Versioneer
+    */_version.py
```

Comment:
Should this also go in .coveragerc?  (Which I’d be +1 on merging into setup.cfg BTW)

Reply:
Probably, these are vendored files with very low hit rates since they cover many continencies. Their coverage (or lack) does not reflect statsmodels.